app/models/user.rb

```diff
@@ -1,16 +1,15 @@
 class User < ApplicationRecord
   has_one_attached :photo
   validates :photo, presence: true
-  has_many :bookings
-  has_many :trips, through: :booking
+  has_many :trips, dependent: :destroy
+  has_many :bookings, dependent: :destroy
+  has_many :joined_trips, through: :bookings, source: :trip
```

Comment:
!!! This is important !!!!
.joined_trips is the new method to access the trips a user joined